python/mxnet/gluon/loss.py

```diff
@@ -101,7 +101,7 @@ def hybrid_forward(self, F, x, *args, **kwargs):
 class L2Loss(Loss):
     r"""Calculates the mean squared error between `pred` and `label`.
```

Comment:
by the same reasoning, this should be swapped as `label` is the subject.

Reply:
same elsewhere.

Reply:
Thanks, good finding!

Reply:
Thanks!